ground_segmentation:
  n_threads: 4            # number of threads to use.
  r_min: 1.0              # minimum point distance.
  r_max: 60               # maximum point distance.
  n_bins: 150             # number of radial bins.
  n_segments: 360         # number of radial segments.
  max_dist_to_line: 0.3   # maximum vertical distance of point to line to be considered ground.
  sensor_height: [1.84, 0.64, 0.66, 0.46]      # sensor height above ground.
  max_slope: 0.3          # maximum slope of a ground line.
  max_fit_error: 0.15      # maximum error of a point during line fit.
  long_threshold: 5.0     # distance between points after which they are considered far from each other.
  max_long_height: 0.3    # maximum height change to previous point in long line.
  max_start_height: 0.3   # maximum difference to estimated ground height to start a new line.
  line_search_angle: 0.3  # how far to search in angular direction to find a line [rad].
  # multi sensors
  extrinsic_trans_left: [0.243332, 0.699051, -1.32942]
  extrinsic_rot_left: [ 0.968392, 0.00840856, -0.0164247, 0.248752 ]

  extrinsic_trans_right: [0.303802, -0.666534, -1.34038]
  extrinsic_rot_right: [0.970389, 0.00652891, -0.00987211, -0.241255]

  extrinsic_trans_rear: [-3.64768, 0.124779, -1.63426]
  extrinsic_rot_rear: [-0.0384245, -0.0183758, -0.00398884, 0.999084]

  # the self point cloud
  x_range_min: -4.0
  x_range_max: 0.5
  y_range_min: -1.2
  y_range_max: 1.2
